enterprise/kernel/src/main/java/org/neo4j/kernel/impl/store/format/highlimit/v30/HighLimitV3_0.java

```diff
@@ -55,7 +55,8 @@
 
     public HighLimitV3_0()
     {
-        super( STORE_VERSION, 7, Capability.DENSE_NODES, Capability.SCHEMA, Capability.LUCENE_5 );
+        super( STORE_VERSION, 7, Capability.DENSE_NODES, Capability.SCHEMA, Capability.LUCENE_5,
+                Capability.FIXED_REFERENCE );
```

Comment:
The condition in `StoreMigrator` was changed from `fromFormat.equals( toFormat )` to only comparing format capabilities. I don't think that's what we want because standard and highlimit format has the same _capabilities_, that doesn't mean they look the same on disk. Introducing new format capabilities, like dense nodes f.ex. is to signal that there will need to be restructuring of the records, i.e. a complete migration even if the record bits on disk would be the same. Format capability wasn't intended to be used for cases like this, so what happens if we revert the condition in `StoreMigrator` and just don't introduce the `FIXED_REFERENCE` capability at all?